ext/bg/js/settings/backup.js

```diff
@@ -120,7 +120,7 @@ async function _onSettingsExportClick() {
 // Importing
 
 async function _settingsImportSetOptionsFull(optionsFull) {
-    return utilIsolate(await utilBackend().setFullOptions(
```

Comment:
I don't think places which use `utilBackend()` have a guarantee that the backend is fully prepared. `setFullOptions` may still need to internally wait for `prepare`, or `utilBackend` may need to wait.

Reply:
Looking at the new update that was pushed as I reviewed, honestly a bigger problem is the design of the settings page, and how none of it waits for initialization currently. Similar to how frontend-initialize.js is the main entry point for content scripts, at some point there should be a main entry point for all settings handler scripts. That's a large change outside the scope of this one though IMO.

Reply:
Do you think it would help if `await yomichan.prepare()` was called from `onReady` in `bg/js/settings/main.js`?

Reply:
Actually looking back at the code, it looks like that's the primary place, I thought there were more places where the scripts initialized themselves. So you're right.

May also be worth it to have utilBackend() do a synchronous check to see if the backend is ready before it returns it, and throws an error if it's not, since that should be a programmer error if that happens.

Reply:
Addressed in https://github.com/FooSoft/yomichan/pull/386/commits/e0edb30efd51ac18167880d77c2dea11c73a1bfc